packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -99,7 +100,8 @@ export interface IEditableTextState {
     value?: string;
 }
 
-const BUFFER_WIDTH = 30;
+const BUFFER_WIDTH_EDGE = 5;
+const BUFFER_WIDTH_IE = 30;
```

Comment:
really??? why do they need _different_ magic numbers? can't we just choose one value that works well enough for both?